src/sidebar/components/test/selection-tabs-test.js

```diff
@@ -1,245 +1,252 @@
 'use strict';
 
-const angular = require('angular');
-
-const util = require('../../directive/test/util');
-
-describe('selectionTabs', function() {
-  const fakeSession = {
-    state: {
-      preferences: {
-        show_sidebar_tutorial: false,
-      },
-    },
-  };
-  const fakeSettings = {
-    enableExperimentalNewNoteButton: false,
+const { shallow, mount } = require('enzyme');
+const { createElement } = require('preact');
+
+const NewNoteBtn = require('../new-note-btn');
+const uiConstants = require('../../ui-constants');
+const SelectionTabs = require('../selection-tabs');
+
+describe('SelectionTabs', function() {
+  // mock services
+  let fakeSession;
+  let fakeSettings;
+
+  // default props
+  const defaultProps = {
+    isLoading: false,
+    isWaitingToAnchorAnnotations: false,
```

Comment:
@LMS007 When I was using this branch and adding in the modifications for moving these to the store I noticed that there don't appear to be any tests in here for isWaitingToAnchorAnnotation= true.